### PR TITLE
Updated VS 2008 Documentation & Gitignore for VS builds

### PR DESCRIPTION
**Pull request by gdwnldsKSC**
I've created a gitignore that ignores the artifacts created by visual studio that aren't relevant to source control (user-local configs, autogenerated intellisense database, etc), as well as updated the documentation with explicit steps required to compile under VS 2008.

### CODE REVIEW

**Comment by gdwnldsKSC** [comment]
Updated with libxml version updates and GPL notification changes in about dialog

File: HydraIRC/version.h [hunk 1]
```diff
@@ -66,4 +66,4 @@
 #endif
 */
 
-#define AUTHORSTRING "Dominic Clifton aka Hydra"
+#define AUTHORSTRING "Dominic Clifton aka Hydra\n Upgrades and newer OS and VS support by gdwnldsKSC"
```
Comment (hydra) [inline HydraIRC/version.h:69]: Understand you'd like some attribute, but not via this.  Feel free to add additional strings and make them appear on the Help/About dialog.

**Review by hydra — CHANGES_REQUESTED** [review]
Can you also remove the `.map` file and add it to `.gitignore`?

All good except for the above and the `AUTHORSTRING` change.  I note that you've already updated the aboutdlg.cpp file, that should be fine right?

**Comment by gdwnldsKSC** [comment]
@hydra Should be good now, made the requested AUTHORSTRING change and updated gitignore to remove map and pdb files from the distro folders. 

Been quite the adventure so far trying to drag this thing kicking and screaming into the future. Lots more to do, but it's shaping up to be a fun challenge.

**Comment by hydra** [comment]
Merged, thanks!

**Pull request merged**